acceptance/setup/pre_suite/15_setup_repos.rb

```diff
@@ -5,6 +5,10 @@ def initialize_repo_on_host(host, os)
     on host, "dpkg -i puppetlabs-release-$(lsb_release -sc).deb"
     on host, "apt-get update"
   when :redhat
+    if options[:type] == 'aio' then
+      on host, "curl -O http://nightlies.puppetlabs.com/puppet-agent-latest/repo_configs/rpm/pl-puppet-agent-latest-el-7-x86_64.repo && cp pl-puppet-agent-latest-el-7-x86_64.repo /etc/yum.repos.d/"
```

Comment:
@kbarber , until we release Puppet 4 compatible versions of puppet-agent and puppetserver we need to pull from nightlies. This will be replaced once we have them in the wild.